modules/creating-recurring-etcd-backups.adoc

```diff
@@ -0,0 +1,259 @@
+[id="creating-recurring-etcd-backups_{context}"]
+== Creating recurring etcd backups
+
+Follow these steps to create automated recurring backups of etcd.
+
+Use dynamically-provisioned storage to keep the created etcd backup data in a safe, external location if possible. If dynamically-provisioned storage is not available, consider storing the backup data on an NFS share to make backup recovery more accessible.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. If dynamically-provisioned storage is available, complete the following steps to create automated recurring backups:
+
+.. Create a persistent volume claim (PVC) named `etcd-backup-pvc.yaml` with contents such as the following example:
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: etcd-backup-pvc
+  namespace: openshift-etcd
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 200Gi <1>
+  storageClassName: standard-csi <2>
```

Comment:
If you omit this field, the default storage class will be used to provision the PV

Reply:
I added a note to the callout for <2> to state that if you omit the field, the default storage class is used.

Reply:
yes but this storage class used no longer exist on 4.16 iiuc

Reply:
I removed the `storageClass` field and its associated callout. Thanks!